client/src/modules/journal/templates/references.link.html

```diff
@@ -0,0 +1,11 @@
+
+<div class="ui-grid-cell-contents">
+  <a 
+    style="text-align:center !important"
+    href="journal/refenrece_transaction/{{row.entity.hrReference}}/{{grid.appScope.languages.key}}"
```

Comment:
Additionally, please change this URL to be `reference_transaction` instead of `refenrece_transaction`.  There is just a minor typo in the spelling of `reference` :+1: